apps/remix-ide/src/app/compiler/compiler-sourceVerifier-fetchAndCompile.js

```diff
@@ -111,7 +115,10 @@ export default class FetchAndCompile extends Plugin {
     }
     try {
       setTimeout(_ => this.emit('compiling', settings), 0)
-      const compData = await compile(compilationTargets, settings)
+      const compData = await compile(
+        compilationTargets,
+        settings,
+        (url, cb) => this.call('contentImport', 'resolveAndSave', url).then((result) => cb(null, result)).catch((error) => cb(error.message)))
```

Comment:
looks like callback hell here ^^.
When you've to a callback in a promise in a callback, you start loosing people XD